src/fieri/app/models/metrics_runner.rb

```diff
@@ -24,4 +24,8 @@ def cookbook_api_response(params)
   def cookbook_version_api_response(params)
     SupermarketApiRunner.new.cookbook_version_api_response(params['cookbook_name'], params['cookbook_version'])
   end
+
+  def source_repo(cookbook_json)
+    cookbook_json.match(%r{(?<=github.com\/)[\w-]+\/[\w-]+}).to_s
```

Comment:
I was initially thinking `GitHubMetric` parent class, but while re-reading the metric definitions, I considered a more generic name like `SourceRepoMetric` (which I agree is still not good) about "checking that something exists in the source repo" and that GitHub just happens to be the first git service we'll support.